flax/linen/__init__.py

```diff
@@ -17,22 +17,26 @@
 
 # pylint: disable=g-multiple-import
 # re-export commonly used modules and functions
-from .activation import (celu, elu, gelu, glu, leaky_relu, log_sigmoid,
-                         log_softmax, relu, sigmoid, soft_sign, softmax,
-                         softplus, swish, silu, tanh, PReLU)
+from .activation import (PReLU, celu, elu, gelu, glu, leaky_relu, log_sigmoid,
```

Comment:
these changes were the result of running a formatter. i figured it's probably easier to send through this change, rather than the linter complaining.